apps/admin_panel/assets/src/omg-page-api/index.js

```diff
@@ -115,13 +140,16 @@ class ApiKeyPage extends Component {
       <KeySectionEwallet>
         <h3>E-Wallet API Key</h3>
         <p>
-        The eWallet API key is used to authenticate an API and allows that specific API to access various user-related functions, e.g. make transfers with the user's wallets, list a user's transactions, create transaction requests, etc.
+          The eWallet API key is used to authenticate an API and allows that specific API to access
```

Comment:
eWallet API Keys are used to authenticate clients and allow them to perform various user-related functions (once the user has been logged in), e.g. make transfers with the user's wallets, list a user's transactions, create transaction requests, etc.